### PR TITLE
Update poll.js

### PR DESCRIPTION
**Pull request by ashyudhajit**
A better poll system lol.Also added pr and tpoll

### CODE REVIEW

**Comment by jaredgrady** [comment]
Can you fix your travis build?

**Comment by ashyudhajit** [comment]
Lol its fine.

**Comment by ashyudhajit** [comment]
Firstly merge this

**Comment by panpawn** [comment]
@ashyudhajit  do you understand what Travis is?

File: chat-plugins/poll.js [hunk 1]
```diff
@@ -1,14 +1,10 @@
-/*
-* Poll chat plugin
-* By bumbadadabum and Zarel.
-*/
```
Comment (panpawn) [inline chat-plugins/poll.js:4]: Why are we removing credits...?

Comment (panpawn) [inline chat-plugins/poll.js:4]: Don't remove credits.

**Comment by ashyudhajit** [comment]
Yes i understand travis.As i updated the code a bit i removed credits....

**Comment by jaredgrady** [comment]
But the code isn't original so you should really leave the credits in. Also if you understand travis then you should have no problem fixing the build.

**Comment by panpawn** [comment]
"Updated the code"

More like copied and pasted other changes.  And you didn't write this poll system, so don't act like you can remove credits.  I suggest you read PS's licensing so that you can understand copy right and fair use.  You can use PS code but you shouldn't remove credits or otherwise try to mark it off as your own when it's not original.

**Comment by ashyudhajit** [comment]
Ok np adding credits.But along them my name too.
Understood?bro i like u don't mind..

**Comment by MasterFloat** [comment]
how dishonest...

**Comment by ashyudhajit** [comment]
U fool. 
Imma not dishonest..

**Comment by ashyudhajit** [comment]
It's not the actual repo. 
Why bother....

**Comment by panpawn** [comment]
"It's not the actual repo.   Why bother...."

You realize there are legal copyrights that protect credits, right?  Legal copyrights, as in, actual real life lawsuits that are a thing when you remove credit from a thing like PS, a MIT licensed project.  

PS!'s licensing can be found here: https://github.com/Zarel/Pokemon-Showdown/blob/master/LICENSE

Please give it a read if you don't think removing credits is a big deal, perhaps learn a few things too.

**Pull request closed without merging**